examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Running the Example
+
+1. Install dependencies `yarn install`
+2. Build source `yarn run compile`
```

Comment:
Hmm, it didn't on my end... I think it does on some other packages because we have postinstall hooks